components/attempt.js

```diff
@@ -10,6 +10,9 @@ class Attempt extends React.Component {
       value: '',
       minLength: 2
     };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleEnter = this.handleEnter.bind(this);
```

Comment:
well, you can see all that using the bindAll function but i guess at the end of the day is just a matter of personal taste.